cpp/docs/TRANSITIONGUIDE.md

```diff
@@ -878,16 +878,34 @@ Therefore, most operations that produce a new output column of strings use a two
 In scatter, the first phase consists of using the `scatter_map` to determine if string `i` in the output will come from `destination` or from `scatter_values` and use the corresponding size(s) to materialize the offsets column and determine the size of the output. Then, in the second phase, sufficient storage is allocated for the output's characters, and then the characters are filled with the corresponding strings from either `destination` or `scatter_values`.
 
 
-### UTF8 
+### `cudf::string_view`
 
-// TODO
+This is the data type for a STRING column like int32_t is the data type for an INT32 column. As it's name implies, this is a read-only object instance that points to device memory inside the strings column. It's lifespan is the same as the column it was created from.
 
-### `cudf::string_view`
+Use the `column_device_view::element` method to access an individual row element. Like any other column, do not call `element()` on a row that is null. 
+```
+   column_device_view d_strings;
+   ...
+   if( d_strings.is_valid(row_index) ) {
+      string_view d_str = d_strings.element<string_view>(row_index);
+      ...
+   }
+```
+A null string is not the same as an empty string. Use the `string_scalar` class if you need to instance of a class object to represent a null string.
+
+The `string_view` contains comparison operators `<,>,==,<=,>=` so they can be used in many cudf functions like `sort` without specific strings code.
+The data for a `string_view` instance is required to be [UTF-8](#UTF-8) and all operators and methods expect this encoding. Unless otherwise, specified
```

Comment:
```suggestion
The data for a `string_view` instance is required to be [UTF-8](#UTF-8) and all operators and methods expect this encoding. Unless documented otherwise,
```